public/app/features/scenes/components/Scene.tsx

```diff
@@ -17,22 +17,20 @@ export class Scene extends SceneObjectBase<SceneState> {
   static Component = SceneRenderer;
   urlSyncManager?: UrlSyncManager;
 
-  onMount() {
-    super.onMount();
+  activate() {
+    super.activate();
     this.urlSyncManager = new UrlSyncManager(this);
   }
 
-  onUnmount() {
-    super.onUnmount();
+  deactivate() {
+    super.deactivate();
```

Comment:
Naming is hard... I don't necessarily like activate/deactivate - maybe onActivate/onDeactivate would speak louder to me. But also I don't have any better proposal for these methods.

Reply:
@dprokop I agree 100% , not 100% happy with this either, but wanted to move away from the mount/unmount naming is for scene objects the same instance can be re-activated